stdlib/2and3/codecs.pyi

```diff
@@ -217,7 +227,49 @@ class StreamReaderWriter(TextIO):
     def seek(self, offset: int, whence: int = ...) -> int: ...
     def __enter__(self: _T) -> _T: ...
     def __exit__(self, typ: Optional[Type[BaseException]], exc: Optional[BaseException], tb: Optional[types.TracebackType]) -> bool: ...
+    def __getattr__(self, name: str) -> Any: ...
+
+    # These methods don't actually exist directly, but they are needed to satisfy the TextIO
```

Comment:
Is there no way to introduce these through inheritance?

Reply:
They're abstract on the base classes, so we need to have them here to make these classes non-abstract. Maybe there's some class in `io` that we could inherit from, but that wouldn't be accurate either since this class doesn't inherit from those classes at runtime.

Reply:
OK, let's keep this (verbose) approach then.